### PR TITLE
harmonising respiratory tract and MCC names

### LINKED ISSUE #2769: rename respiratory epithelial cells to 'respiratory tract epithelial ...'
Respiratory alone is potentially confusing as only the terminal airways and aveoli have respiratory function.  

image

### PR DESCRIPTION
**Pull request by dosumis**
respiratory --> respiratory tract
MCC all now have multiciliated in name
Some work may still be needed to ensure all synonyms in place

Fixes #2769 #2768

### CODE REVIEW

**Comment by dosumis** [comment]
Have set to draft for now as need to add synonyms for all changes + we may want to more radically rename in some cases.

@aleixpuigb - wondering about how we work with Lung nomenclature efforts & MIRO boards on this?  Maybe update all and leave a note about the change making clear it is nomenclature not meaning that has shifted?

**Comment by dosumis** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Comment by Caroline-99** [comment]
Everything looks good. I have added any missing synonyms for all the changes. @aleixpuigb could you please have a look at the changes when you have time ? Thank you

File: src/ontology/cl-edit.owl [hunk 1]
```diff
@@ -31143,12 +31151,13 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25311230") rdfs:comment
 AnnotationAssertion(rdfs:label obo:CL_4030033 "valve endothelial cell")
 EquivalentClasses(obo:CL_4030033 ObjectIntersectionOf(obo:CL_0000115 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000946)))
 
-# Class: obo:CL_4030034 (respiratory ciliated cell)
```
[the remark this answers is not in the document]
Comment (Caroline-99) [reply src/ontology/cl-edit.owl]: I agree. Have added it as a broad synonym to the term respiratory tract multiciliated cell

**Comment by dosumis** [comment]
Why broad?

**Comment by aleixpuigb** [comment]
> Why broad?

Because there are some respiratory cells that have only one cilia (primary cilia). Ciliated respiratory cell would be a broader class that encompasses both multiciliated respiratory cells and primary ciliated respiratory cells.

**Comment by dosumis** [comment]
Don't all cells have a primary cilium?

**Comment by Caroline-99** [comment]
> Don't all cells have a primary cilium?

Are you suggesting changing it to an exact synonym instead?

**Comment by gpryhuber** [comment]
I think the primary cilium that most human cells have, serving a non-motile sensory function, are very different from the numerous, highly motile cilia that we are referring to in "multiciliated cells" of the tracheobronchial epithelium. Their structure is readily distinguishable.

**Comment by Caroline-99** [comment]
#gogoeditdiff

**Review by aleixpuigb — APPROVED** [review]

**Pull request merged**